parse/ast.go

```diff
@@ -704,6 +671,112 @@ func (v Variant) validate() error {
 	return nil
 }
 
+type Option struct {
+	Node
+
+	Value      Value // Literal or Variable
+	Identifier Identifier
+}
+
+func (o Option) String() string { return fmt.Sprintf("%s = %s", o.Identifier, o.Value) }
+func (o Option) validate() error {
+	if err := o.Identifier.validate(); err != nil {
+		return fmt.Errorf("option.%w", err)
+	}
+
+	if o.Value == nil {
+		return errors.New("option: value is required")
+	}
+
+	if err := o.Value.validate(); err != nil {
+		return fmt.Errorf("option.%w", err)
+	}
+
+	return nil
+}
+
+type MarkupType int
+
+const (
+	Unspecified MarkupType = iota
+	Open
```

Comment:
Shouldn't these be prefixed with 'Markup'?

Reply:
After a quick look, it seems prefixes are not necessary